assets/performanceprofile/configs/99-default-rps-mask.conf

```diff
@@ -1,5 +1,5 @@
-# We should apply the rule on the virtual interfaces of the host, because of the RPS mask that would be consulted,
-# is the one on the RX side of the veth in the host.
+# Apply the RPS mask on the virtual interfaces of the host by default, becasue
```

Comment:
@Tal-or  please note that there was a rename here, is that correct ?

Reply:
yes why not?